### DISCUSSION TITLE
OpenUp

### DISCUSSION THREAD

**Discussion by Horryportier** [charmbracelet/inspo#41] [Ideas]
**Is your feature request related to a problem? Please describe.**
My problem is its annoying to go thought directories to open something like config file or project. So my idea is to build  a  simple TUI to list projects and files i often use and open them with editor of choice.

**Describe the solution you'd like**
- list of files/dir to select and open.
- choice of editor like vim/vscode/etc.
- searching option.

**Include projects that are similar in nature**

**Comment by nkxxll** [comment]
Do you already have a repo? I would love to contribute!

**Comment by Horryportier** [comment]
I will add one when I made some basic prototype

**Comment by Horryportier** [comment]
added basic capabilities
- reads list items from json
- opens file upon selection. Tested in vim/neovim
#### To implement 
- Adding/delting items
- choice of editor 

## link to repo: 
https://github.com/Horryportier/openup
